### PR TITLE
Clear image list cache

### PR DESCRIPTION
**Pull request by the-last-pastafarian**

### CODE REVIEW

File: server/lib/field_hub/file_store.ex [hunk 1]
```diff
@@ -4,6 +4,7 @@ defmodule FieldHub.FileStore do
   @valid_file_variants Application.compile_env(:field_hub, :valid_file_variants)
   @index_cache_name Application.compile_env(:field_hub, :file_index_cache_name)
   @index_cache_expiration_ms 1000 * 60 * 60 * 24
+  # @index_cache_expiration_ms 3000
```
Comment (dersmon) [inline server/lib/field_hub/file_store.ex:7]: What did you try here? This line should probably be deleted at the end.

File: server/lib/field_hub_web/live/project_show_live.ex [hunk 2]
```diff
@@ -112,6 +117,75 @@ defmodule FieldHubWeb.ProjectShowLive do
     {:noreply, assign(socket, :new_password, password)}
   end
 
+  # def handle_event("delete_cache", %{"project" => project} = _values, socket) do
+  def handle_event("delete_cache", _values, %{assigns: %{project: project}}=socket) do
+    new_cache = "Delete or update cache in this handle_event"
+    cache_index = FieldHub.FileStore.file_index(project)
```
Comment (dersmon) [inline server/lib/field_hub_web/live/project_show_live.ex:123]: For the ticket you do not need to create a new cache, that is already handled by the `FileStore` internally.

The only thing that has to be triggered from the interface is the FileStore's function to clear the existing cache. If you want to have a look how the file index looks like, you could for example use the Desktop client to push new images into the repository.

**Review by dersmon — COMMENTED** [review]
It looks like you branched from the project deletion, that is why those changes currently also appear here in the diff (which is between the new branch and `master`). I ignored that for now.

File: server/lib/field_hub/file_store.ex (hunk 1 above)
Comment (the-last-pastafarian) [reply server/lib/field_hub/file_store.ex:7]: Yes, I wanted to see what happened without wait one day.

**Pull request merged**